src/components/SubCategoryDetails/SubCategoryDetails.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import getSubCategorySectionId from '../../helpers/getSubCategorySectionId';
+import CityColumnChart from '../CityColumnChart/CityColumnChart';
+import { INDICATORS } from '../../constants';
+import style from './SubCategoryDetails.scss';
+
+const SubCategoryDetails = (props) => {
+  const sc = props.subCategory;
+
+  const heroChart = sc.charts.find(chart => (
+    chart.indicatorIds.includes(props.heroIndicatorId)));
+
+  const otherCharts = sc.charts.filter(chart => (
+    !chart.indicatorIds.includes(props.heroIndicatorId)));
+
+  return (
+    <div
+      key={sc.name}
```

Comment:
`key` can go. It's only needed inside a loop (where it's in the actual context of the array, you don't need it if a component was originally rendered as the output of a loop, if that makes sense)